src/JobSeekerJobListCard.jsx

```diff
@@ -15,9 +15,11 @@ const JobSeekerJobListCard = ({listing, seeDetailsVisibility}) => {
   const closeDate = listing.Job.close_date.slice(0, 10);
   const job_id = listing.job_id;
 
+  let jobDetails = {};
```

Comment:
Might want to change this to state variable like company how company name is

Reply:
Updatedd